tests/handling.py

```diff
@@ -1,78 +1,97 @@
 import unittest
 
-from ..tools.handler import handler
+from ..handler import handler
 
-class TestHandlerFunction(unittest.TestCase):
 
+class TestHandlerFunction(unittest.TestCase):
     def test_handle_bodyless_event(self):
-        event = {
-            "random": "metadata",
-            "without": "a body"
-        }
+        event = {"random": "metadata", "without": "a body"}
 
         response = handler(event)
 
         self.assertIn("error", response)
         error = response.get("error")
 
         self.assertEqual(
-            error.get("message"), 
-            "No evaluation data supplied in request body.")
+            error.get("message"),
+            "No grading data supplied in request body.",
+        )
 
     def test_non_json_body(self):
-        event = {
-            "random": "metadata",
-            "body": "{}}}{{{[][] this is not json."
-        }
+        event = {"random": "metadata", "body": "{}}}{{{[][] this is not json."}
 
         response = handler(event)
 
         self.assertIn("error", response)
         error = response.get("error")
 
         self.assertEqual(
-            error.get("message"),
-            "Request body is not valid JSON.")
+            error.get("message"), "Request body is not valid JSON."
+        )
+
+    def test_eval(self):
+        event = {
+            "random": "metadata",
+            "body": {"response": "hello", "answer": "world!", "params": {}},
+            "headers": {"command": "eval"},
+        }
+
+        response = handler(event)
+
+        self.assertEqual(response.get("command"), "eval")
+        self.assertIn("result", response)
 
-    def test_grade(self):
+    def test_eval_no_params(self):
         event = {
             "random": "metadata",
-            "body": {
-                "response": "hello",
-                "answer": "world!",
-                "params": {}
-            },
-            "headers": {
-                "command": "grade"
-            }
+            "body": {"response": "hello", "answer": "world!"},
+            "headers": {"command": "eval"},
         }
 
         response = handler(event)
 
-        self.assertEqual(response.get("command"), "grade")
+        self.assertEqual(response.get("command"), "eval")
         self.assertIn("result", response)
 
-    def test_grade_no_params(self):
+    def test_handler_evals_by_default(self):
         event = {
             "random": "metadata",
-            "body": {
-                "response": "hello",
-                "answer": "world!"
-            }
+            "body": {"response": "hello", "answer": "world!"},
         }
 
         response = handler(event)
 
-        self.assertEqual(response.get("command"), "grade")
+        self.assertEqual(response.get("command"), "eval")
+
+    def test_preview(self):
```

Comment:
A lot of these tests share the same assertions. Consider using parametrized tests (given you are using `unittest`, a good option is probably https://github.com/wolever/parameterized). 

Also, strive to make the test name as descriptive as possible; don't be afraid of long names -within reason, obv, although if your test is so large that its name takes more than a few words to be described accurately, then the problem lies in the scope of the test, not in its name.